CopilotKit/packages/runtime/src/graphql/resolvers/copilot.resolver.ts

```
@@ -250,373 +251,420 @@ export class CopilotResolver {
+                        !(
+                          e.type === RuntimeEventTypes.TextMessageEnd &&
+                          e.messageId == event.messageId
+                        ),
+                    ),
+                    // filter out any other message events or message ids
+                    filter(
+                      (e) =>
+                        e.type == RuntimeEventTypes.TextMessageContent &&
+                        e.messageId == event.messageId,
+                    ),
+                  );
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)